src/containers/App/index.js

```diff
@@ -21,6 +21,10 @@ import style from './app.scss'
 import transitionStyles from './transitions.scss'
 import 'rc-tooltip/assets/bootstrap.css?raw'
 
+if (isFeatureEnabled('tournament')) {
```

Comment:
@andre-meyer This is a good example why we should split configuration into runtme/build time